package/MDAnalysis/analysis/align.py

```diff
@@ -232,20 +233,21 @@ def rotation_matrix(a, b, weights=None):
     :func:`rmsd` calculates the RMSD between *a* and *b*; for fitting a whole
     trajectory it is more efficient to use :func:`rms_fit_trj`. A complete fit
     of two structures can be done with :func:`alignto`. """
-    if weights is not None:
-        # weights are constructed as relative to the mean
-        weights = np.asarray(weights, dtype=np.float64) / np.mean(weights)
 
     a = np.asarray(a, dtype=np.float64)
     b = np.asarray(b, dtype=np.float64)
     N = b.shape[0]
 
+    if weights is not None:
+        weights = np.asarray(weights, dtype=np.float64)
+
     rot = np.zeros(9, dtype=np.float64)
     rmsd = qcp.CalcRMSDRotationalMatrix(a.T, b.T, N, rot, weights)
+
     return np.matrix(rot.reshape(3, 3)), rmsd
 
 
-def alignto(mobile, reference, select="all", mass_weighted=False,
+def align_to(mobile, reference, select="all", mass_weighted=False,
```

Comment:
> I thought you told me to do this to follow PEP8?

PEP8 isn’t gospel ;-) – it’s generally more important (especially with names) not to break the API unless you really, really have to!

(Or your Apple and then you give a damn.)